pkg/compose/compose.go

```diff
@@ -320,3 +320,7 @@ func (s *composeService) RuntimeVersion(ctx context.Context) (string, error) {
 	return runtimeVersion.val, runtimeVersion.err
 
 }
+
+func (s *composeService) isDesktopIntegrationActive() bool {
```

Comment:
👍🏻

Reply:
I refactored other places in the code that also did this validation